### ISSUE TITLE
OpenAPI 3.0

### ISSUE THREAD

**Issue by SWSAmor** [DapperDox/dapperdox#63]
Would you support that in the near future?
https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md

Thx.

**Comment by zxchris** [comment]
Yes definitely - we're downstream of go-openapi parser, which handles spec parsing for us. And internally we're spec neutral (ish), so short answer: yes!

**Comment by zxchris** [comment]
Just waiting on go-openapi

**Comment by SWSAmor** [comment]
Thank you. Waiting patiently. I love DapperDox. :-)

**Comment by alexjomin** [comment]
any update please ?

**Comment by hotgazpacho** [comment]
Maybe depends on go-openapi/spec#21 ?

**Comment by polarweasel** [comment]
That go-openapi issue is still open, and it looked like they were taking the discussion to their Slack channel. @zxchris are you aware of any progress?

**Comment by david-l-riley** [comment]
Same question here. If it helps move things along, I don't mind contributing some work. We really could use this for our own docs.

**Comment by quazgar** [comment]
Upstream is closed now.
